index.html

```diff
@@ -345,6 +346,7 @@ <h6 class="h-contact"> contact me</h6>
 
         </form>
```

Comment:
- [ ] You did great here, :+1: Your form functions as expected, but there are some styling issues you need to fix before moving to the next milestone.

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] As part of the requirement is to replicate the given design as much as you can, Looking at the screenshot from the Figma design below, you are missing some background images, I'll suggest you get all the images and update your UI accordingly.
- [ ] Also the position of your top background image is different from the given design, the background image is behind the text and it makes it hard to see and read the text, I'll suggest you update this image position too. :+1: :smile: 

- - Your design:
![Selection_059](https://github.com/hetrox8/portfolio1/assets/107632780/0602e621-6666-4b6f-86ca-ea525104646a)

- - Figma design:
![Selection_058](https://github.com/hetrox8/portfolio1/assets/107632780/49e24fea-a5a6-4b1c-997a-0dfe73fa4120)